src/components/ScholarshipsMadeSimpleGrid.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { Container } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+import PropTypes from 'prop-types';
+import logopic from '../img/blank.png';
+import LoginButton from './LoginButton';
+
+function ContentDiplay(user) {
+  if (user === 0) {
+    return (
+      <div>
+        <Typography variant="h6" component="h6">
```

Comment:
No I was not aware of the overlain variant, but I did play around with it and the font size seems to be a bit too small but It could be my opinion but if you agree I can change it back to how I originally had it, right now im using this line of code.
```javascript
<Typography variant="overline" component="h6">
  {tab === 'students' ? 'students' : 'community contributor'}
</Typography>
```